app/public/data/dashboards/cc21f006274175d0.json

```diff
@@ -56,7 +56,7 @@
           "lat": 35.19310377935969
         }
       },
-      "id": "Biwa-NLK@1689236387909",
+      "id": "NLK_lake_biwa-NLK@1689236387909",
```

Comment:
Are we sure we need the double indicator+poi notation? I guess it should work in any case but would no longer be necessary, right?